.travis.yml

```diff
@@ -60,12 +60,16 @@ jobs:
     # the `platforms` defined in `kitchen.yml`
     - env: INSTANCE=repositories-debian-10-master-py3
     - env: INSTANCE=preferences-debian-10-master-py3
+    - env: INSTANCE=unattended-debian-10-master-py3
     # - env: INSTANCE=repositories-ubuntu-1804-master-py3
     # - env: INSTANCE=preferences-ubuntu-1804-master-py3
+    # - env: INSTANCE=unattended-ubuntu-1804-master-py3
     - env: INSTANCE=repositories-debian-9-2019-2-py3
     - env: INSTANCE=preferences-debian-9-2019-2-py3
+    - env: INSTANCE=unattended-debian-9-2019-2-py3
     - env: INSTANCE=repositories-ubuntu-1804-2019-2-py3
     - env: INSTANCE=preferences-ubuntu-1804-2019-2-py3
+    - env: INSTANCE=unattended-ubuntu-1804-2019-2-py3
```

Comment:
I'd like to suggest a change to this matrix, to minimise Travis concurrency issues:

* https://travis-ci.com/saltstack-formulas/apt-formula/builds/141394833 -- currently in this PR (1+9 jobs)
* https://travis-ci.org/myii/apt-formula/builds/626591663 -- proposed (1+4 jobs)
  - https://github.com/myii/apt-formula/blob/60_0/.travis.yml#L58-L76 -- the definition for this
  - Each suite gets tested twice, once in the `debian-10` multi-suite job and then one specific job per suite

Reply:
@myii Should I do something here, or it is going to be discussed by the team first?

Reply:
@fzipi We can proceed with the changes and then get this merged.  Usually, I'm able to add commits to a PR before merging but I'm not able to here.  Did you deselect `Allow edits from maintainers`?  Either way, please include these changes here:

* https://github.com/myii/apt-formula/commit/af24de6d41a949a5d23195486583a71eb17cf223

If you can also rebase on the latest version of this formula, we can avoid any issues with the `train` gem causing problems with `debian-10`.

Reply:
Rebased and pushed including that. Allow edits from maintainers is enabled there .... weird...